### PR TITLE
Create Posts “2020-11-10-elasticsearch_and_redis_deprecation_notice”

### PR DESCRIPTION
**Pull request by karareinsel**
Automatically generated by Netlify CMS

# Security considerations
none (publishing an existing deprecation notice as a blog post)

### CODE REVIEW

File: _posts/2020-11-10-elasticsearch_and_redis_deprecation_notice.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: Elasticsearch and Redis deprecation notice
+date: November 10, 2020
+---
+As part of our reliability and stability enhancements to cloud.gov, the legacy Redis 3.2 and Elasticsearch 5.6 offerings from cloud.gov will be deprecated as of April 5, 2021. In accordance with our deprecation policy, provisioning of new instances will be disabled on December 6, 2020 and on April 5, 2021 we will shut down all active instances. This work is an important step in moving cloud.gov towards a more stable and reliable platform by moving away from a bespoke Kubernetes instances to AWS-hosted services. Learn more about our deprecation policy here:<https://cloud.gov/docs/technology/responsibilities/#deprecation-policy>
```
Comment (ccostino) [inline _posts/2020-11-10-elasticsearch_and_redis_deprecation_notice.md]: Small word adjustment to account for a grammar correction:

```suggestion
As part of our reliability and stability enhancements to cloud.gov, the legacy Redis 3.2 and Elasticsearch 5.6 offerings from cloud.gov will be deprecated as of April 5, 2021. In accordance with our deprecation policy, provisioning of new instances will be disabled on December 6, 2020 and on April 5, 2021 we will shut down all active instances. This work is an important step in moving cloud.gov towards a more stable and reliable platform by moving away from a bespoke Kubernetes cluster to AWS-hosted services. Learn more about our deprecation policy here:<https://cloud.gov/docs/technology/responsibilities/#deprecation-policy>
```

**Review by ccostino — COMMENTED** [review]
Thanks, @karareinsel!  There was one small grammatical change I noticed but otherwise this is 👍 from me.

**Review by ccostino — APPROVED** [review]
Thanks, @karareinsel!

**Comment by bengerman13** [comment]
@karareinsel the security considerations check is pretty strict about formatting - the line "security considerations" needs to start with a `#` and the text needs to be on the next line, with an optional blank line:
```
# security considerations

considerations go here
```

(I updated your comment so this'll go through, just FYI for the future 😄 )

**Pull request merged**